src/pages/en/migrate.md

```diff
@@ -42,6 +42,10 @@ Astro no longer supports components or JSX expressions in Markdown pages by defa
 
 To make migrating easier, a new [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) can be used to re-enable previous Markdown features.
 
+### `<Markdown />` Component Removed
+
+If you were previously using Astro’s built-in `<Markdown />` component, this has been moved to a separate package. You will now need to install `@astrojs/markdown` and update your imports accordingly. For more details, see [the `@astrojs/markdown` README](https://github.com/withastro/astro/tree/main/packages/markdown/component).
```

Comment:
Thanks @matthewp ! 

@delucis , I'm assuming that this `@astrojs/markdown` component does NOT get automatically added to our Integrations page?

- Do we *want* a legacy package on our integrations page, so that we don't *need* to send people externally to the README but can instead have a page in Docs? Or, do we specifically NOT want this showing up in Docs, because we don't want people using it unless they really have to?

- Do we *want* to take this opportunity here in the migration guide to encourage people to transition to MDX, or simply say, "Here's where the new package is?"

Reply:
I think we don't want people using it unless they really have to.

Agree that it's a good opportunity to push people towards MDX, I'll add that.